Letters/src/main/java/org/Marta/LowercaseText.java

```diff
@@ -0,0 +1,10 @@
+package org.Marta;
+
+import java.util.Locale;
+
+public class LowercaseText implements TextFormatter {
```

Comment:
Jak nie masz pomysłu na nazwę implementacji to pierw spróbuj od rozszerzenia nazwy interfejsu ;)

Interfejs: TextFormatter klasa LowerCaseTextFormatter


"LowercaseText" nie brzmi jak metoda, ktora dostarcza formater

Reply:
ok, przyznam że często odpowiednie nazywanie tych klas, metod i testów sprawia mi więcej problemów niż wymyślenie logiki działania tych programików xD